### PR TITLE
implemented around filter

### LINKED ISSUE #6: Implement around filters
Documentation: [Overpass_QL#Relative to other elements (around)](http://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL#Relative_to_other_elements_.28around.29)

Implement filters:

```
(around:radius)
(around:radius,latitude,longitude)
```

Note: the optional `[.input_set]` modifier is intentionally left out. Input sets here and in other parts of the library will be supported in a later stage.

Goal: add methods to `OverpassFilterQuery` with the following signatures:

``` java
public OverpassFilterQuery around(double radius)
```

and

``` java
public OverpassFilterQuery around(double radius, double lat, double lon)
```

### PR DESCRIPTION
**Pull request by wolfhardfehre**
- fixes #6

### CODE REVIEW

**Comment by zsoltk** [comment]
Hey @wolfhardfehre , thanks for the commit! I'll add some comments on the changes before I can merge, please review them.

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.2.2'
```
Comment (zsoltk) [inline build.gradle]: While this is a valid change, it violates separation of concerns: it's not necessary for implementing around filters and it's not related to the original problem. As such, it must not be part of this PR, but rather a separate one.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Wed Sep 16 18:17:45 CEST 2015
+#Tue Nov 15 23:34:48 CET 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```
Comment (zsoltk) [inline gradle/wrapper/gradle-wrapper.properties]: The same applies here, this should go into a separate PR.

File: sample/build.gradle [hunk 3]
```diff
@@ -4,7 +4,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:2.1.2"
+        classpath 'com.android.tools.build:gradle:2.2.2'
```
Comment (zsoltk) [inline sample/build.gradle]: This should go into a separate PR as well (all of them three touching gradle config can however go into the same one).

File: library/src/test/java/hu/supercluster/overpasser/library/query/UsageExamplesTest.java [hunk 4]
```diff
@@ -123,4 +123,43 @@ public void testMultipleFilterQueriesWithGlobalBoundingBoxSearch() throws Except
...
+                .filterQuery()
+                .node()
+                .amenity("parking")
+                .tagNot("access", "private")
+                .around(50.5)
+                .prepareNext()
+                .way()
+                .amenity("parking")
+                .tagNot("access", "private")
+                .around(
```
Comment (zsoltk) [inline library/src/test/java/hu/supercluster/overpasser/library/query/UsageExamplesTest.java]: I know Android Studio flattens these methods to the same indentation level, but please observe how other code is formatted. For greater readability, methods between `.filterQuery()` and `.endQuery()`, and `.node()` / `.way()` / etc. and `.prepareNext()` must be one level deeper regarding indentation. Please follow the style of the rest of the file.

**Comment by wolfhardfehre** [comment]
@zsoltk sorry that it took me a while ;)

**Comment by SajeOne** [comment]
Is there any chance of this being merged? I really like the library but I require this feature, thanks.

**Comment by zsoltk** [comment]
@wolfhardfehre can you please fix the merge conflicts so that I can merge?

**Comment by wolfhardfehre** [comment]
@zsoltk Sorry can't anymore... unfortunately delete the fork already and now when I fork it again I can't checkout the branch ;( ... if it's very urgent, maybe someone else could copy and paste my changes, having no time to work on it right now.
